### PR TITLE
Update main.rs

### PR DESCRIPTION
**Pull request by MayoDeNocheAntagonist**

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -1,3 +1,4 @@
 fn main() {
     println!("Hello, world!");
+    println!("Nya, :3~);
```
Comment (Peelshine) [inline src/main.rs:3]: ermmmmmmm sdyntax error

Comment (MayoDeNocheAntagonist) [reply src/main.rs:3]: _**nuh uh**_

Comment (Peelshine) [reply src/main.rs:3]: well, ill be damned

Comment (Peelshine) [reply src/main.rs:3]: merging

**Review by Peelshine — APPROVED** [review]

**Pull request merged**